Build/scripts/init_build.ps1

```diff
@@ -105,6 +106,13 @@ if (-not (Test-Path $flavorBuildIncompleteFile)) {
         | Out-File $flavorBuildIncompleteFile -Encoding Ascii
 }
 
+$PogoConfig = "False"
```

Comment:
I think we should pass as argument which flavor we want to pogo.
This is okay for the time being, but we should consider changing it